scripts/update_production.sh

```diff
@@ -38,6 +38,7 @@ fi
 echo "Disabling config_readonly."
 touch disable-config-readonly
 
+./vendor/bin/phing redis-flush
```

Comment:
We'll have to chain with AND operator the commands so that the failure of one makes the whole process fail.